packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# 48.5.0 - March 5, 2020
+
+## Fixed
+
+#### salesforcedx-vscode-apex
+
+- Remove error from the Apex LSP when a user types 'Page.' for completion results ([PR #2054](https://github.com/forcedotcom/salesforcedx-vscode/pull/2054))
```

Comment:
Thank you adding this

Reply:
Could rephrase as:
Fix error in completion results of `Page.` in Apex LSP